README.md

```diff
@@ -232,6 +226,106 @@ int main(int argc, char* argv[]) {
    imwrite("image.bmp", image, true);
 }
 ```
+
```

Comment:
Perhaps one question that we could clarify is if this case and also inpainting require special models or you can just plug regular SD model. Latter will be beneficial of course..